libraries/rush-lib/src/cli/test/__snapshots__/CommandLineHelp.test.ts.snap

```diff
@@ -128,8 +128,9 @@ exports[`CommandLineHelp prints the help for each action: build 1`] = `
 "usage: rush build [-h] [-p COUNT] [--timeline] [-t PROJECT] [-T PROJECT]
                   [-f PROJECT] [-o PROJECT] [-i PROJECT] [-I PROJECT]
                   [--to-version-policy VERSION_POLICY_NAME]
-                  [--from-version-policy VERSION_POLICY_NAME] [-v] [-c]
-                  [--ignore-hooks] [-s] [-m]
+                  [--from-version-policy VERSION_POLICY_NAME]
+                  [--subspace SUBSPACE_NAME] [-v] [-c] [--ignore-hooks] [-s]
```

Comment:
Why does `rush build` specify `a Rush subspace to be installed`?  That seems incorrect